cli/util/git.go

```diff
@@ -66,3 +70,31 @@ func IsGitFetchJobsSupported() bool {
 	}
 	return isGitFetchJobsSupported(out.String())
 }
+
+// IsValidCommitHash checks hash format.
+func IsValidCommitHash(hash string) (bool, error) {
+	if len(hash) < MinCommitHashLength {
+		return false, fmt.Errorf("the hash must contain at least 7 characters")
+	}
+	return regexp.MatchString(`^[0-9a-f]+$`, hash)
+}
+
+// IsPullRequest checks pull-request format.
+func IsPullRequest(input string) bool {
+	input = strings.ToLower(input)
+	if strings.HasPrefix(input, "pr/") {
+		prNum := strings.Split(input, "/")[1]
+		isPullRequest, _ := regexp.MatchString(`^[0-9]+$`, prNum)
+		return isPullRequest
+	}
+	return false
+}
+
+// GetPullRequestID get num in pr/<num> format
+func GetPullRequestID(request string) string {
```

Comment:
Maybe we can use single function like `GetPullRequestID(request string) (string, bool)`? Instead of two functions. Also,  `IsPullRequest` does the same split.

Reply:
fixed, now only 1 function